src/python/pants/option/config.py

```diff
@@ -192,14 +192,10 @@ def has_section(self, section):
     return self.configparser.has_section(section)
 
   def has_option(self, section, option):
-    return (self.configparser.has_option(section, option) or
-            self.configparser.has_option(self._DEFAULT_SECTION, option))
+    return self.configparser.has_option(section, option)
 
   def get_value(self, section, option):
-    if self.configparser.has_option(section, option):
-      return self.configparser.get(section, option)
-    else:
-      return self.configparser.get(self._DEFAULT_SECTION, option)
+    return self.configparser.get(section, option)
```

Comment:
In the options system yes, but our config parser has changed behavior, and we used it in at least one other place: `contrib/android/src/python/pants/contrib/android/keystore/keystore_resolver.py`

Probably not a big deal though, since that class's docstring says

```
A Keystore config is an .ini file with valid syntax as parsed by Python's ConfigParser.
```

So this is actually more correct behavior than before.